etc/config/c++.defaults.properties

```diff
@@ -1,5 +1,5 @@
 # Default settings for C++
-compilers=&gcc:&clang
+compilers=&gcc:&clang:&edg:&edggnu
```

Comment:
I'd suggest not having this by default: most folks won't have EDG installed and this means every user by default will try and find the EDG compilers (this is why we only have gcc and clang here in the defaults, and not all the 200 other types of compilers we support...)